resources/lang/nld.ini

```diff
@@ -1,46 +1,111 @@
+; wand tool
+tool.wand = "Toverstaf"
+tool.wand.lore.1 = "Klik met de linkermuisknop op een blok om de positie 1 van een selectie in te stellen"
+tool.wand.lore.2 = "Klik met de rechtermuisknop op een blok om de positie 2 van een selectie in te stellen"
+tool.wand.lore.3 = "Gebruik //togglewand om van functionaliteit te wisselen"
+tool.wand.disabled = "De toverstaf is uitgeschakeld. Gebruik //togglewand om het opnieuw in te schakelen"
+tool.wand.setenabled = "Het gereedschap Toverstaf is nu{%0}!"
+; debug tool
+tool.debug = "Foutopsporing gereedschap"
+tool.debug.lore.1 = "Klik met de linkermuisknop op een blok om informatie te krijgen"
+tool.debug.lore.2 = "zoals de naam en schadewaarden van een blok"
+tool.debug.lore.3 = "Gebruik //toggledebug om van functionaliteit te wisselen"
+tool.debug.disabled = "Het foutopsporing gereedschap is uitgeschakeld. Gebruik //toggledebug om het opnieuw in te schakelen"
+tool.debug.setenabled = "Het foutopsporing gereedschap is nu {%0}!"
+; flood tool
+ui.flood.title = "Opvullings menu"
+ui.flood.options.limit = "Maximum aantal blokken"
+ui.flood.options.blocks = "Blokken"
+ui.flood.options.blocks.placeholder = "Blokken onderbroken door semicolons"
+ui.flood.options.label.infoapply = "Klik op de "Submit" knop om het op te sturen"
+; brush tool
 ui.brush.title = "Kwast menu"
-; ui brush select
-ui.brush.select.title = "Selecteer een kwast type"
-ui.brush.select.type.sphere = "Bol"
-ui.brush.select.type.cylinder = "Cilinder"
-ui.brush.select.type.cuboid = "Balk"
-ui.brush.select.type.clipboard = "Klembord"
-; ui brush settings
+ui.brush.content = "Kwast hoofdmenu"
+ui.brush.create = "Maak nieuw"
+ui.brush.getsession = "Krijg sessie kwast"
+ui.brush.edithand = "Verander kwast in hand"
+; brush settings
 ui.brush.settings.title = "{%0} kwast instellingen"
-; ui brush options
+; brush options
 ui.brush.options.blocks = "Blokken"
 ui.brush.options.blocks.placeholder = "Bijvoorbeeld 1:1,2,tnt,log:12"
 ui.brush.options.diameter = "Diameter"
 ui.brush.options.width = "Breedte"
 ui.brush.options.height = "Hoogte"
 ui.brush.options.depth = "Diepte"
-ui.brush.options.flags = "Voegen opties toe?"
-; ui flags
-ui.flags.keepexistingblocks = "Behoudt bestaande blokken"
-ui.flags.keepair = "Behoudt lucht"
-ui.flags.hollow = "Hol"
-ui.flags.natural = "Natuurlijk"
-; ui brush sphere
-; ui brush cylinder
-; ui brush cuboid
-; ui brush clipboard
-; ui flood
-ui.flood.title = "Opvullings menu"
-ui.flood.options.limit = "Maximum aantal blokken"
-ui.flood.options.blocks = "Blokken"
-ui.flood.options.blocks.placeholder = "Blokken onderbroken door semicolons"
-ui.flood.options.label.infoapply = "Klik op de "Submit" knop om het op te sturen"
\ No newline at end of file
+ui.brush.options.flags = "Opties toevoegen?"
+; language
+ui.language.title = "Select language"
+ui.language.label = "Set the language of your session. If your language is not available, you may translate the plugin on GitHub!"
+ui.language.dropdown = "Select a language"
```

Comment:
These strings aren't translated. Are they meant to be?

Reply:
Indeed, they must be translated (maybe a logic mistake by me - maybe it should default to the server default language, but we need the translations anyways)

Reply:
Indeed, they must be translated (maybe a logic mistake by me - maybe it should default to the server default language, but we need the translations anyways)